frontend/dev-mode/src/components/InputCodeSnippets.vue

```diff
@@ -0,0 +1,173 @@
+const session = inject<Ref<Session>>('session', ref(DEFAULT_SESSION))
+const parameters = inject<Ref<Parameters>>('parameters', ref(DEFAULT_PARAMS_VALUES))
+
+const pythonSnippet = computed(() => `import openai
+
+client = openai.OpenAI(
+  base_url="${apiUrl}/v1",${
+    parameters.value.api_key ? '\n  api_key="'+parameters.value.api_key+'"' : ''
+  }
+)
+
+completion = client.chat.completions.create(
+  model="${currentModel.value}",
+  messages=[
+    { "role": "system", "content": "${session.value.prompt}" },${
+      (session.value.transcript as Array<any>).map(([role, [entry]]) => {
+        if (role.toLowerCase() === '{{user}}') {
+          return `\n    { "role": "user", "content": "${entry.content}" }`
+        }
+      })
+      .filter(Boolean)
+      .reverse()
+      .join(',')
+    }
+  ],
+  ${
+    Object.entries(parameters.value).map(([key, value]) => {
+      return `${key}="${value}"`
+     }).join(',\n  ')
+  }
+)
+
+for chunk in completion:
+  if chunk.choices[0].delta.content is not None:
+    print(completion.choices[0].delta.content, end="")
+`)
+
+const nodeSnippet = computed(() => `import OpenAI from 'openai';
+
+const openai = new OpenAI({
+  baseURL: '${apiUrl}/v1',${
+    parameters.value.api_key ? '\n  apiKey: \''+parameters.value.api_key+'\'' : ''
+  }
+})
+
+async function main() {
+  const completion = await openai.chat.completions.create({
+    model: "${currentModel.value}",
+    messages=[
+      { role: 'system', content: '${session.value.prompt}'},${
+      (session.value.transcript as Array<any>).map(([role, [entry]]) => {
+        if (role.toLowerCase() === '{{user}}') {
+          return `\n      { role: 'user', content: '${entry.content}' }`
+        }
+      })
+      .filter(Boolean)
+      .reverse()
+      .join(',')
+    }
+    ],
+    ${
+      Object.entries(parameters.value).map(([key, value]) => {
+        return `${key}="${value}"`
+      }).join(',\n    ')
+    }
+    stream: true
+  })
+
+  for await (const chunk of completion) {
+    process.stdout.write(chunk.choices[0]?.delta?.content || '')
+  }
+}
+
+main();
+`)
+
+const shSnippet = computed(() => `invoke_url='${apiUrl}/v1/chat/completions'
+${
+parameters.value.api_key
+  ? 'authorization_header=\'Authorization: Bearer '+parameters.value.api_key+'\''
+  : ''
+}
+accept_header='Accept: application/json'
+content_type_header='Content-Type: application/json'
+
+data=$'{
+  "messages": [
+    { "role": "system", "content": "${session.value.prompt}" },${
+    (session.value.transcript as Array<any>).map(([role, [entry]]) => {
+      if (role.toLowerCase() === '{{user}}') {
+        return `\n    { "role": "user", "content": "${entry.content}" }`
+      }
+    })
+    .filter(Boolean)
+    .reverse()
+    .join(',')
+    }
+  ],
+  "model": "${currentModel.value}",
+  "stream": true,
+  ${
+    Object.entries(parameters.value).map(([key, value]) => {
+      return `"${key}"": ${value}`
```

Comment:
There is an extra `"` that is breaking the snippets. You can see that the json on shell Tab is malformed